utils/symbols.stanza

```diff
@@ -63,76 +63,76 @@ public pcb-symbol altium-power-arrow-sym :
   name = "ALTIUM-POWER-ARROW"
   pin p[0] at unit-point(0.0, 0.0)
 
-  unit-line([[0.0, 0.0], [1.27, 0.0]])
-  unit-triangle([1.27, 0.0], [2.794, 0.0], 1.524)
-  unit-val([2.5, -0.5])
+  unit-line([[0.0, 0.0], [0.5, 0.0]])
+  unit-triangle([0.5, 0.0], [1.1, 0.0], 0.6)
+  unit-val([1.0, -0.2])
 
   preferred-orientation = PreferRotation([1])
 
 public pcb-symbol altium-power-circle-sym :
   name = "ALTIUM-POWER-CIRCLE"
   pin p[0] at unit-point(0.0, 0.0)
 
-  unit-circle([3.0 / 2.0 * 1.27, 0.0], 1.27 / 2.0)
-  unit-line([[0.0, 0.0], [1.27, 0.0]])
-  unit-val([2.5, -0.5])
+  unit-circle([3.0 / 2.0 * 0.5, 0.0], 0.5 / 2.0)
+  unit-line([[0.0, 0.0], [0.5, 0.0]])
+  unit-val([1.0, -0.2])
   
   preferred-orientation = PreferRotation([1])
 
 public pcb-symbol altium-power-bar-sym : 
   name = "ALTIUM-POWER-BAR"
   pin p[0] at unit-point(0.0, 0.0)
 
-  val ex = 3.0 / 2.0 * 1.27
+  val ex = 3.0 / 2.0 * 0.5
   unit-line([[0.0, 0.0], [ex, 0.0]])
-  unit-line([[ex, -1.27 / 2.0], [ex, 1.27 / 2.0]])
-  unit-val([0.5, 0.5])
+  unit-line([[ex, 0.5 / 2.0], [ex, 0.5 / 2.0]])
```

Comment:
First one should be -0.5 and not 0.5
This is why in the visualiser the line is the wrong direction